blocks/test/fixtures/core__author___hide.html

```diff
@@ -0,0 +1 @@
+<!-- wp:core/author {"hideBio":true,"hideAvatar":true,"hideName":true} /-->
```

Comment:
The `wp:core/` is no longer necessary, and can be seen as stripped in the serialized version.

Reply:
I changed wp:core/author to wp:author, but unfortunately, some of our PHP tests started failing (possible to see in our CI). I probably misunderstood the change :(